rclpy/test/test_messages.py

```diff
@@ -48,14 +48,6 @@ def test_unicode_string(self):
         self.node.destroy_publisher(pub)
 
     def test_different_type_raises(self):
-        # TODO(bmarchi): When a publisher is destroyed for opensplice,
-        # the topic from the previous test is kept around and is cached, so
-        # the new publisher can't be created because opensplice checks
-        # if the parameters of the created topic are the same as the one
-        # that has in memory. It's expected that if any participant
-        # is not subscribed/publishing to a topic, this is destroyed.
-        # Revert the topic name to 'chatter' once proper topic destruction
```

Comment:
I actually do not think we need to do this, since `chatter_different_message_type` sounds more appropriate for this test.